packages/ezflow_extension/ezextension/ezflow/classes/ezflowoperations.php

```diff
@@ -73,6 +73,7 @@ public static function updateBlockPoolByBlockID( $block, $publishedBeforeOrAt =
         $fetchClassOptions->iniFile = 'block.ini';
         $fetchClassOptions->iniSection = $block['block_type'];
         $fetchClassOptions->iniVariable = 'FetchClass';
+        $fetchClassOptions->handlerParams = array(new eZFlowFetchParameters($block));
```

Comment:
should we follow old legacy coding standars here? If so, some spaces are missing.